client/src/components/dashboard/TemplateSelector.js

```diff
@@ -170,15 +170,17 @@ const TemplateSelector = (props) => {
                 <Col className="layout-display">
                   <Form.Check
                     checked={formInputs.layout === 3}
-                    inline
                     type="radio"
                     name="layout"
-                    label="Layout 3"
+                    label="layout 3"
                     value={3}
                     className="form-radio-text"
                     onChange={updateForm}
+
+
```

Comment:
Not hundred percent sure the reason for removing `inline` and changing the `label`, but if you're changing it I suppose you might have missed the other two (Layout 1 and Layout 2)?
Btw, can remove the empty lines in 179 and 180 ig :)